upup/pkg/fi/cloudup/azuretasks/disk.go

```diff
@@ -116,20 +121,25 @@ func (*Disk) RenderAzure(t *azure.AzureAPITarget, a, e, changes *Disk) error {
 	name := *e.Name
 
 	disk := compute.Disk{
-		Location: to.StringPtr(t.Cloud.Region()),
-		DiskProperties: &compute.DiskProperties{
+		Location: to.Ptr(t.Cloud.Region()),
+		Properties: &compute.DiskProperties{
 			CreationData: &compute.CreationData{
-				CreateOption: compute.Empty,
+				CreateOption: to.Ptr(compute.DiskCreateOptionEmpty),
 			},
 			DiskSizeGB: e.SizeGB,
 		},
+		SKU: &compute.DiskSKU{
+			Name: to.Ptr(compute.DiskStorageAccountTypesStandardSSDLRS),
```

Comment:
is this not configurable? It feels like it should be

Reply:
I will add the option in a future PR.